### DISCUSSION TITLE
Seattle Fire Station Locator

### DISCUSSION THREAD

**Discussion by zeyany2** [jakobzhao/geog495#50] [Final Project]
https://github.com/zeyany2/geog495_finalproject
![application](https://user-images.githubusercontent.com/46511898/145441194-25868c0e-3794-4873-9fba-78c97664d719.png)

For my **Geog495** final project, I build  a Seattle FireStation locator using Mapbox GL JS. The goal for this project is to help people who live in Seattle area get to know where the fire stations are and can use the application to find them.

**Comment by VaneRaklan** [comment]
I like the vibrancy of the color palette and the icon size for the fire stations, I feel like no matter my zoom level I can very easily see where everything is.
Your application might benefit from either more information about each fire station (in the popup) or something to indicate interactivity; something like changing the color of the fire station that you're hovering over. If you really wanted to get fancy you could add something to help people find the closest fire department to wherever they are looking at. It lends itself to your projects idea about helping people get to know the fire stations in their area.

**Comment by kevography** [comment]
The format of the UI is very clean and I like how legible the map is.
My main concern is that the web map doesn't exactly match its stated purpose. It would be very helpful to add some form of geolocator that would allow the user to search their current location. I think an additional functionality that would really elevate this would be an estimate for each fire station to the user's location.

**Comment by zhuxiaoyu1019** [comment]
I like how you make it very easy for people to understand what you're trying to convey. I appreciate the simplicity and ease of use of the application.
A cool feature you may consider is that you could add a buffer to show the area of coverage of fire stations when the user clicks on one of the icons.

**Comment by jacobf19** [comment]
The icons on this map are really cool and I like how simple and friendly the look of the webpage is. I think this map would be improved with some sort of functionality or user customization in order to see something unique and different based on the users needs. The map is effective in its current state, but is lacking from not having anything to change on the user level.

**Comment by notzulkifli** [comment]
The idea is great, I like that the interactivity on the side-panel is usable! The icon design is perfect to represent the fire stations in seattle as well!

My one concern is that we aren't able to sort the data or find the nearest fire station from each firestation. For example, One firestation from one street is ____ miles away so that way the user is able to identify where each fire station is, and how far it could be from where they are staying or how far it is from another fire station.

**Comment by mhan799** [comment]
Hi, I think overall this is a great map. It is easy for users to understand and the choice of the marker is appropriate too. I think you can also improve this map by providing more information about each fire station in the sidebar, also, maybe switch the location of the title and the address.

**Comment by VkSath00** [comment]
I really like the way this map looks, its straight to the point and the icon for indicating the fire stations is great. Maybe a way to improve this application could be to include the ability for the user to visualize the distance in miles from the searched location and you could also add a little more info in the sidebar about the stations.
